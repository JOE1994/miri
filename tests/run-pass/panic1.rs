fn main() {
    panic!("Miri panic!");
}
