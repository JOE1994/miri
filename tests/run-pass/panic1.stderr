thread 'main' panicked at 'Miri panic!', $DIR/panic1.rs:2:5
